_layouts/report.md

```diff
@@ -82,6 +81,12 @@
                             <div class="center"><h5 class="light no-margin">Patches Tarball</h5></div>
                             <br>
                             <div class="center"><a href="{{ page.tarball }}">Download</a></div>
+                            <br>
+                            <div class="center">
+                                <p style="word-wrap: break-word;" >
+                                    <strong>SHA-256</strong>: {{page.sha256sum}}
```

Comment:
IMO the best design is that you have an abbreviation (say, the first seven letters) displayed, and a button beside it, which people can click to copy the hash. Example: https://github.com/coala/projects/pull/689/commits (see the hash and copy icon)

Reply:
But this is debatable; feel free to discuss here

Reply:
Ok, I will create this design and will post its screenshot with some `js` included in `report.md` to implement that `copy` feature.

Reply:
![image](https://user-images.githubusercontent.com/35761292/48213785-96444b80-e3a4-11e8-884e-ad2e3fc2756c.png)
Is this ok or you want me to implement same as GitHub has done ?

Reply:
@li-boxuan  Also, I have added some Javascript to implement the `copy` feature, so should I create a new file or remain that `js code` in the `html` document only. The existing `JS` files include app related functionality and no `JS` file has been created to handle or manipulate the HTML elements so either we have to create a new file or keep that code in the `html code` in the end of body.

Reply:
Can you push what you have so far so that reviewers apart from me can comment on it?

Reply:
Ok

Reply:
![image](https://user-images.githubusercontent.com/35761292/48255386-87a47580-e432-11e8-9ed2-9397c4ffaf9c.png)